src/applications/check-in/day-of/containers/withLoadedData.jsx

```diff
@@ -24,6 +24,7 @@ const withLoadedData = Component => {
     const { isSessionLoading, router, setSessionData } = props;
     const selectCheckInData = useMemo(makeSelectCheckInData, []);
     const checkInData = useSelector(selectCheckInData);
+    const { getCurrentToken } = useSessionStorage('health.care.check-in');
```

Comment:
random thought, is there a good place to store this string in one place?

Reply:
Good point, it may make sense to encapsulate the actual strings in the hook and and pass in a flag or something. I can dig into this one a little more.